### DISCUSSION TITLE
3.6.2

### DISCUSSION THREAD

**Discussion by patientAtGithub** [nalexandru/BlenderCompat#56] [General]
Hello nalexandru,

Thank you for helping users remain productive with Blender on **win7** OS. I hope you are having a pleasant summer of course. 

A new version of blender (3.6.2) was released a few days ago I am sure you know, and I would like to ask if you will have the time to spare to continue offering compiled releases.

Thank you so much again for your precious time and essential help to not just me but all of us.

All the best.
-patientAtGithub

**Comment by dancingshoes** [comment]
Thanks again for the Win 7 support! Life saver!

**Comment by nalexandru** [comment]
Thank you for letting me know there is a new version available. I've uploaded 3.6.2 and 3.3.10.
I will continue to make binaries available, however I am in a busy period at work currently and as a result there will be some delays.
